.travis.yml

```diff
@@ -0,0 +1,34 @@
+os: linux
+dist: bionic
+language: bash
+services: docker
+
+branches:
+  only:
+  - master
+
+env:
+  - VERSION=4.7
+
+jobs:
+  include:
+    - stage: build x86_64 container image
+      arch: amd64
+      script:
+        - docker build --build-arg RELEASE_VER=$VERSION -t openshift-install-powervs:v1.1.2-ocp$VERSION-$(uname -m) -f images/Dockerfile .
```

Comment:
The version should not be hardcoded.... v1.1.2 should be the tag version which we are going to push.